x-pack/test/detection_engine_api_integration/security_and_spaces/group10/import_rules.ts

```diff
@@ -1131,8 +1209,90 @@ export default ({ getService }: FtrProviderContext): void => {
           expect(body.errors.length).to.eql(0);
         });
 
+        it('should import a non-default-space 7.16 rule with a connector made in the non-default space', async () => {
```

Comment:
nit: split in sections by rule type. This test and the next one can be combined in a group `non-default-space 7.16 rule with a connector made in the non-default space` with tests
- `should be imported into the non-default space`
- `should be imported into the default space`

Reply:
please find 9799b5f8aa5807e37c5334ff6aa02f0c948ebf98